fw/t/unit/test_http_parser.c

```diff
@@ -905,59 +783,6 @@ TEST(http_parser, casesense)
  */
 TEST(http_parser, hdr_token_confusion)
 {
-	FOR_REQ("GET / HTTP/1.1\r\n"
-		"Accept: textK/html\r\n"
-		"Connection: closekeep-alive\n"
-		"Pragma: no-cacheX, fooo \r\n"
-		"Cache-Control: max-staleno-cache, no-storeno-store\r\n"
```

Comment:
The test was about checking the state machine at the end of term `no-store` and now it seems the test is missed. 

Basically, I wouldn't replace any tests with an equal cases - there is no sense to run less tests. It's not a big deal if tests are from the same euqivalence class. Let's replace tests only for refactoring, when we run exactly the same test, but in bit different way, e.g. using the new macros.

Reply:
`"Cache-Control: no-storeno-store\r\n"` is not OK? It's done in `TEST_COMMON` in `"Cache-Control:" directive directive` case.

Reply:
I didn't find any function or macro with name `TEST_COMMON` - what it is?

Reply:
Sorry, in this commit it's TEST_NO_ARGUMENT, line 1112.